setup.py

```diff
@@ -19,8 +19,8 @@
       py_modules=['Chandra.axTime3', 'Chandra.Time'],
       version='3.20',
       zip_safe=False,
-      test_suite="Chandra.test_Time",
-      packages=['Chandra'],
-      package_dir={'Chandra': 'Chandra'},
+      test_suite="Chandra.Time.test_Time",
+      packages=['Chandra', 'Chandra/Time'],
```

Comment:
Originally I just had the thought of getting everything in `Chandra.Time` to actually live within the installed `Chandra/Time/` directory.  That is nice and clean.  But in practice it is true that this is forced to live in a namespace package directory anyway, so perhaps it doesn't matter?  If you can get things working OK then I'm fine with leaving it (but as mentioned it would be *nicer* to have it organized in a tidy way).